pkg/server/test/property_value_history_test.go

```diff
@@ -131,3 +134,120 @@ var getPropertyValueBoolean testServerScenarioFn = func(t *testing.T) *testScena
+	propTimeSeries := testdata.GetIoTSitewiseTimeSeries(t, testDataRelativePath("describe-time-series.json"))
+
+	mockSw.On("BatchGetAssetPropertyValueHistoryPageAggregation", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(&propVals, nil)
+	mockSw.On("DescribeAssetPropertyWithContext", mock.Anything, mock.Anything).Return(&propDesc, nil)
+	mockSw.On("DescribeTimeSeriesWithContext", mock.Anything, mock.Anything).Return(&propTimeSeries, nil)
+
+	query := models.AssetPropertyValueQuery{
+		BaseQuery: models.BaseQuery{
+			AwsRegion:     testdata.AwsRegion,
+			PropertyAlias: testdata.TurbinePropWindSpeedAlias,
+		},
+	}
+
+	return &testScenario{
+		name:   "PropertyValueHistoryFromAliasResponseHappyCaseTable",
+		mockSw: mockSw,
+		queries: []backend.DataQuery{
+			{
+				QueryType:     models.QueryTypePropertyValueHistory,
+				RefID:         "A",
+				MaxDataPoints: 100,
+				Interval:      1000,
+				TimeRange:     timeRange,
+				JSON:          testdata.SerializeStruct(t, query),
+			},
+		},
+		goldenFileName: "property-history-values-from-alias-table",
+		handlerFn: func(srvr *server.Server) backend.QueryDataHandlerFunc {
+			return srvr.HandlePropertyValueHistory
+		},
+	}
+}
+
+var getPropertyValueHistoryFromAliasCaseTimeSeries testServerScenarioFn = func(t *testing.T) *testScenario {
+
+	mockSw := &mocks.SitewiseClient{}
+
+	propVals := testdata.GetIoTSitewisePropHistoryVals(t, testDataRelativePath("property-history-values.json"))
+	propDesc := testdata.GetIotSitewiseAssetProp(t, testDataRelativePath("describe-asset-property-avg-wind.json"))
+	propTimeSeries := testdata.GetIoTSitewiseTimeSeries(t, testDataRelativePath("describe-time-series.json"))
+
+	mockSw.On("BatchGetAssetPropertyValueHistoryPageAggregation", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(&propVals, nil)
+	mockSw.On("DescribeAssetPropertyWithContext", mock.Anything, mock.Anything).Return(&propDesc, nil)
+	mockSw.On("DescribeTimeSeriesWithContext", mock.Anything, mock.Anything).Return(&propTimeSeries, nil)
+
+	query := models.AssetPropertyValueQuery{
+		BaseQuery: models.BaseQuery{
+			ResponseFormat: "timeseries",
+			AwsRegion:      testdata.AwsRegion,
+			PropertyAlias:  testdata.TurbinePropWindSpeedAlias,
+		},
+	}
+
+	return &testScenario{
+		name:   "PropertyValueHistoryFromAliasResponseHappyCaseTimeSeries",
+		mockSw: mockSw,
+		queries: []backend.DataQuery{
+			{
+				QueryType:     models.QueryTypePropertyValueHistory,
+				RefID:         "A",
+				MaxDataPoints: 100,
+				Interval:      1000,
+				TimeRange:     timeRange,
+				JSON:          testdata.SerializeStruct(t, query),
+			},
+		},
+		goldenFileName: "property-history-values-from-alias-timeseries",
+		handlerFn: func(srvr *server.Server) backend.QueryDataHandlerFunc {
+			return srvr.HandlePropertyValueHistory
+		},
+	}
+}
+
+var getPropertyValueBooleanFromAlias testServerScenarioFn = func(t *testing.T) *testScenario {
```

Comment:
I think we should move away from how the existing tests were written.
Here's my suggestion: https://github.com/grafana/iot-sitewise-datasource/pull/183/files#diff-3054cdb6a1d1aa7888cd1b888d5e0d9745012593389ed18e7728d97167806a93R222